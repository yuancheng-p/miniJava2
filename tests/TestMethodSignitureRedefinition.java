package tutu.titi.toto;

//class A can pass the test
public class A {
    Int a1 = 1;
    int[] a = { 3,4,5 };
    Bool a2 = false;
    String a6 = "jsdjkndjk\njdsdjkf";

    public void fooA();
    public void fooA(int toto);

}

//class B can't pass the test
public class B {
    int fooB(int tata);
    void fooB(int toto);
}
