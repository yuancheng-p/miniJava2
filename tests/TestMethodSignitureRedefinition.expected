successfull parsing
package tutu.titi.toto
public class A extends Object {
  Int a1 = 1;
  int[] a = {3,4,5};
  Bool a2 = false;
  String a6 = "jsdjkndjk\njdsdjkf";
  void fooA()  {
  }
  void fooA(int toto)  {
  }
}

public class B extends Object {
  int fooB(int tata)  {
  }
  void fooB(int toto)  {
  }
}

start typing
check add and methods:A:fooA
check add and methods:A:fooA
check add and methods:B:fooB
